Userland/Applications/ThemeEditor/VisionSimulator.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, Antonio Di Stefano <tonio9681@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/Palette.h>
+
+namespace ThemeEditor {
+enum class VisionType {
+    NON_IMPAIRED,
+    PROTANOPIA,
+    PROTANOMALY,
+    DEUTERANOPIA,
+    DEUTERANOMALY,
+    TRITANOPIA,
+    TRITANOMALY,
+    ACHROMATOPSIA,
+    ACHROMATOMALY,
```

Comment:
Please use `PascalCase` for enum members

Reply:
Fixed